server/config/contactForm.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    CONCERT_MAIL: ['concert@kfet-insa.fr', 'technique@kfet-insa.fr'],
```

Comment:
Maybe one day, user can manage this ?

Reply:
Everything here will be pass by env variable (and one day directly by the kapp)
Because a env variable is only a String, we will have to create a new util function to split a string ino an array.

Reply:
See slack!